component_setup_kingfisher.ipynb

```diff
@@ -68,6 +68,9 @@
         "# Don't show connection string after execute.\n",
         "%config SqlMagic.displaycon = False\n",
         "\n",
+        "# Set SqlMagic/prettytable style\n",
+        "%config SqlMagic.style = '_DEPRECATED_DEFAULT'\n",
+        "\n",
```

Comment:
You can remove this. Just make sure to use the latest ocdskingfishercolab (0.3.14).